### ISSUE TITLE
Unable to load verified moderators.

### ISSUE THREAD

**Issue by zababurinsv** [OpenBazaar/openbazaar-desktop#1918]
Brief Description: 

Operating System (OS and version): Linux Ubuntu 20.04
OpenBazaar version (OpenBazaar Server v0.14.6): 
Hardware: 

Reproducible (Always / Almost Always / Sometimes / Rarely / Couldn't Reproduce): 

Steps to reproduce:
1. Install Openbazaar-go v0.14.6
2. set in configuration:
    * "JSON-API"
         * "Authenticated": true,
         * "Password": ""
         * "Username": ""
3. clone openbazaar-desktop
4 npm install
5 npm run start
6 Set in menu:
    * server: 127.0.0.1
    * username
    * password
    * port 4002
    * ssl: off
7 connect to server
8 ob://search

Observed Behavior:
I'am getting error `Unable to load verified moderators.`
Click Retry to try again, or Continue to skip loading the verified moderators. Reason for failure: 522
Expected Behavior:
What is this error ? And how can to avoid it?

Additional info (links, images, etc go here):

![image](https://user-images.githubusercontent.com/22741813/112741346-a36e2f00-8f8d-11eb-85b5-361db0f44395.png)

**Comment by Lew-Rockwell-Fan** [comment]
I'm just guessing here, but:
Did you click 'continue'? If so, what happened? Did you ever see anything for sale? I had this same error message. 'Retry' never had any effect. When I clicked 'continue' I was able to create a silly test listing, but I never saw any other listings.  At first I assumed it wasn't working, then it occurred to me that this is **_exactly what I would expect if the program worked fine, but nobody else was running it at the time I was_**. After all, it's p2p, so I assume a store is only visible when the owner is online and running the program. **_The problem here may be that we don't have critical mass of people using the program in an effective manner._**  I assume that any store owner needs to leave the program, or at least the server if they have the 2 part version, up and running most of time, preferably 24/7.  If nobody is running one, no stores will be seen.  Presumably moderators are people who have volunteered to watch for illegal offers, like aging neutron bombs (hey, gotta keep that stock rotated), and are empowered to remove those stores. The same applies to them. If none are signed on at the moment, there won't be any. I'm just guessing.  Anyway, I'm going to test part of that guess soon by putting up a serious store (the test I tried was silly) and trying to keep it running as near 24/7 as I can.  I suggest anyone reading this try to do the same. Right now, I'm doing something that is tying up most of my system resources for maybe 2 more days. Then I'll do it.  In the long run, for a store owner, the best way to do this would probably be a dedicated Raspberry Pi, so there would be little if any down time.
